pkgs/development/web/nodejs/nodejs.nix

```diff
@@ -34,7 +38,7 @@ in stdenv.mkDerivation {
 
   name = "nodejs-${version}";
 
-  configureFlags = concatMap sharedConfigureFlags (builtins.attrNames deps) ++ [ "--without-dtrace" ] ++ extraConfigFlags;
+  configureFlags = concatMap sharedConfigureFlags (builtins.attrNames deps) ++ [ "--without-dtrace" ] ++ extraConfigFlags ++ [ npmFlag ];
```

Comment:
Perhaps `++ npmFlags` and have `npmFlags = lib.optional ...`, so that there isn't that strange empty string added?